data/projects/f/filecoin-project.yaml

```diff
@@ -7,138 +7,7 @@ social:
   twitter:
     - url: https://twitter.com/Filecoin
 github:
-  - url: https://github.com/filecoin-project/actors-utils
```

Comment:
@zechim-dev Do you have any thoughts on how to resolve the conflicts between this project and other projects that have repositories inside this GitHub organization? we discussed this on the call but I wasn't sure how you wanted to handle it